src/preface.md

```diff
@@ -1 +1,70 @@
+<p style="text-align:center;"><img src="./assets/esp-rs.svg" width="25%"></p>
+
 # Preface
+
+Welcome to a comprehensive guide to using the `esp-rs` ecosystem, created for embedded Rust development on Espressif products. This book serves as a practical resource for developers wishing to utilize the capabilities of Rust in `no_std` environments when working with Espressif hardware platforms.
+
+## Who This Book Is For
+
+This book is intended for people with some experience in Rust and assumes basic knowledge of embedded development and electronics. The expected target user of our ecosystem and reader of this book has a basic understanding of concepts such as cross-compilation, common digital interfaces (UART, SPI, I2C), memory-mapped peripherals and interrupts. For those without prior experience, we recommend first reading the [Scope of This Book][prerequisites] and [Resources][resources] sections to get up to speed.
```

Comment:
```suggestion
This book is intended for people with some experience in Rust and assumes basic knowledge of embedded development and electronics. Readers of this book are expected to have a basic understanding of concepts such as cross-compilation, common digital interfaces (UART, SPI, I2C), memory-mapped peripherals and interrupts. For those without prior experience, we recommend first reading the [Scope of This Book][prerequisites] and [Resources][resources] sections to get up to speed.
```

This sounds a bit awkward, maybe something like I've suggested (or whatever else you come up with) would be a bit more natural?